tools/cdk-build-tools/lib/package-info.ts

```diff
@@ -126,6 +126,13 @@ export interface CDKBuildOptions {
    */
   pre?: string[];
 
+  /**
+   * An optional command (formatted as a list of strings) to run after building
+   *
+   * (Typically a code generator)
+   */
+  post?: string[];
+
   /**
```

Comment:
What's this, and why is it needed?

Can't this just be appended to the 'build' script in `package.json'?

```
"build": "cdk-build && <post command>"
```

Reply:
pkglint enforces that the "build" script is always "cdk-build" and if we want "pre" (or "post") actions, they are modeled separately. we can do it differently but that's how it is right now.

Reply:
Right, actually this is not strictly needed anymore. Decided to leave it since i think its a nice feature to have.